From f02567772103238f0503fa9a26d6802955839148 Mon Sep 17 00:00:00 2001
Message-ID: <f02567772103238f0503fa9a26d6802955839148.1719691458.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1719691458.git.sharlatanus@gmail.com>
References: <cover.1719691458.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 18 Jun 2024 19:00:40 +0100
Subject: [PATCH 17/26] gnu: Add go-github-com-errata-ai-regexp2.

* gnu/packages/golang-xyz.scm (go-github-com-errata-ai-regexp2): New variable.

Change-Id: If9af0a91da1d7a08767dea4614ece9fd620a143a
---
 gnu/packages/golang-xyz.scm | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/gnu/packages/golang-xyz.scm b/gnu/packages/golang-xyz.scm
index af7860781d..6f6167972f 100644
--- a/gnu/packages/golang-xyz.scm
+++ b/gnu/packages/golang-xyz.scm
@@ -1500,6 +1500,30 @@ (define-public go-github-com-elliotchance-orderedmap
 for @code{Set}, @code{Get}, @code{Delete} and @code{Len}.")
     (license license:expat)))
 
+(define-public go-github-com-errata-ai-regexp2
+  (package
+    (inherit go-github-com-dlclark-regexp2)
+    (name "go-github-com-errata-ai-regexp2")
+    (version "1.7.0")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/errata-ai/regexp2")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "0p28af5c7dn4knnksl9dxjb44cicsmadzb8kwzyyf20kr7hrq53q"))))
+    (build-system go-build-system)
+    (arguments
+     (list
+      #:import-path "github.com/errata-ai/regexp2"))
+    (home-page "https://github.com/errata-ai/regexp2")
+    (description
+     (string-append (package-description go-github-com-dlclark-regexp2)
+                    "  This package is a fork of dlclark/regexp2 providing a
+more similar API to regexp."))))
+
 (define-public go-github-com-expr-lang-expr
   (package
     (name "go-github-com-expr-lang-expr")
-- 
2.41.0

